### PR TITLE
Add SpriteManager.disableDepthWrite property

### PR DESCRIPTION
**Pull request by Popov72**

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-babylonjs.com
```
Comment (deltakosh) [inline CNAME]: Why this change?

**Comment by Popov72** [comment]
> Only checking for the CNAME

Was testing something on my side, did not see it had been committed, sorry.
